packages/amplify-cli-core/src/overrides-manager/override-skeleton-generator.ts

```diff
@@ -137,7 +137,7 @@ export const generateAmplifyOverrideProjectBuildFiles = (backendDir: string, src
 /**
  * this method generates the tsconfig file template for overrides
  */
-export const generateTsConfigForProject = (srcResourceDirPath: string, destDirPath: string): void => {
+export const generateTsConfigforProject = (srcResourceDirPath: string, destDirPath: string): void => {
```

Comment:
```suggestion
export const generateTsConfigForProject = (srcResourceDirPath: string, destDirPath: string): void => {
```

Reply:
this is the breaking change that @jhockett was referring to above. I am going to keep the old version